modules/arcspanRtdProvider.js

```diff
@@ -0,0 +1,71 @@
+import { submodule } from '../src/hook.js';
+import { mergeDeep } from '../src/utils.js';
+import {loadExternalScript} from '../src/adloader.js';
+
+/** @type {string} */
+const MODULE_NAME = 'realTimeData';
+const SUBMODULE_NAME = 'arcspan';
+
+/** @type {RtdSubmodule} */
+export const arcspanSubmodule = {
+  name: SUBMODULE_NAME,
+  init: init,
+  getBidRequestData: alterBidRequests,
+};
+
+function init(config, userConsent) {
+  if (typeof config.params.silo === 'undefined') {
+    return false;
+  }
+  var scriptUrl;
+  if (config.params.silo === 'test') {
+    scriptUrl = 'https://localhost:8080/as.js';
+  } else {
+    scriptUrl = 'https://silo' + config.params.silo + '.p7cloud.net/as.js';
```

Comment:
please handle publisher hosted js here, or cases where the publisher has already loaded the js before or in parallel with prebid.js to improve performance. Please provide instructions on this, as making your js dependent on prebid.js means less auctions will contain your enrichments

Reply:
@patmmccann We don't currently support publisher-hosted JS. Is that a requirement here? I implemented the performance changes. I also added instructions to the documentation for the publisher to reference our onboarding documentation to deploy the tag outside of Prebid.js. Is that sufficient?

Reply:
that works, ty!